app/controllers/applicants_controller.rb

```diff
@@ -150,33 +150,16 @@ def unassigned
   end
 
   def assign_craftsman
-    applicant_id = params[:applicant_to_assign]["id"]
-    chosen_crafter = params[:applicant_to_assign]["chosen_crafter"]
-    puts "chosen_crafter = #{chosen_crafter}"
-    puts "applicant_id = #{applicant_id}"
-    applicant = repo.applicant.find_by_id(applicant_id)
-    steward = repo.craftsman.find_by_email(ENV['STEWARD'])
-
-    if automatically_assigned?
-      ApplicantDispatch::Dispatcher.new(applicant, steward).assign_applicant
-    else
-      ApplicantDispatch::Dispatcher.new(applicant, steward).assign_applicant_specific(chosen_crafter)
-    end
-    redirect_to(unassigned_applicants_path, notice: "Assigned #{applicant.name} to #{applicant.assigned_craftsman}")
-  end
+    applicant = repo.applicant.find_by_id(params[:applicant_to_assign][:id])
```

Comment:
dig could work for this... `params.dig(:applicant_to_assign, :id)` I think it helps with nil handling, but either syntax is fine